src/services/db/GitHubService.ts

```diff
@@ -541,7 +544,7 @@ export default class GitHubService {
   async updateTree(
     sessionData: UserWithSiteSessionData,
     githubSessionData: GithubSessionData,
-    { gitTree, message }: { gitTree: any; message: any }
+    { gitTree, message }: { gitTree: unknown; message: unknown }
```

Comment:
are we able to get more specific than unknown here?

Reply:
Good catch! Modified it in 8f15d84de12b5550a58fba2e0046a7a628c28d53